tests/rust/function_args.toml

```diff
@@ -0,0 +1 @@
+language = "C"
```

Comment:
This should work regardless of the language, right? I think this should be removed, that way we test both the C and C++ codepath.

Reply:
Right!